src/components/JaegerToolbar/index.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from 'react';
```

Comment:
Is there a reason to name that file `index.tsx` instead of `JaegerToolbar.tsx` ?

Reply:
To make the imports more cleaner

import { JaegerToolbar} from 'components/JaegerToolbar';

I saw other projects that they use this like patternfly-react. We should have these things in the Code Guidelines.

Reply:
There is not problem about we can set the same name and create a index inside components with all the exports and will be the same